src/evergreen-updater.js

```diff
@@ -0,0 +1,130 @@
+// @flow
+'use strict'
+
+/*::
+
+export type ParseUpdateOptions = {
+  data: {
+    available: boolean,
+    compatible: boolean,
+    snapshot: string,
+    url: string
+  }
+}
+
+export type IonicDeploy = {
+  download: (string, Function, Function) => void,
+  redirect: (string) => void,
+  parseUpdate: (string, ParseUpdateOptions, Function, Function) => void,
+  deploy: (string, Function, Function) => void,
+}
+
+*/
+
+export default class EvergreenUpdater {
+  /*:: ionicDeploy: any */
```

Comment:
You have defined the `IonicDeploy` type above but not using it here?

Reply:
ah, good pickup. thats me being sick